cli.go

```diff
@@ -60,13 +82,35 @@ func main() {
 	}
 }
 
-func PromptDiscord(action string) *DiscordInstall {
+func PromptDiscord(action, dir, branch string) *DiscordInstall {
+
+	if branch != "" {
+		for _, discord := range discords {
+			install := discord.(*DiscordInstall)
+			if install.branch == branch {
+				return install
+			}
+		}
+		fmt.Println("Discord" + branch + " is not installed on your pc")
+		os.Exit(1)
+	} else if *&dir != "" {
+		if discord := ParseDiscord(*&dir, branch); discord != nil {
+			return discord
+		}
```

Comment:
```suggestion
		} else {
			fmt.Println("The specified directory is not a valid Discord install")
			os.Exit(1)
		}
```

Reply:
after this change, you will now have obsolete dir logic at the bottom so please remove it